content/overview/workflow/process.md

```diff
@@ -0,0 +1,209 @@
+
+If you decide to use the _lens correction_ module, it should be enabled at the start of your edit, before you adjust exposure, since vignetting correction can alter the overall brightness of your image.
+
+### reduce noise / retain detail
+
+At a pixel level there is a trade-off to be made between the retention of fine details and the reduction/removal of sensor noise. In most situations, a small amount of noise is perfectly acceptable and will not be noticeable except when you zoom in to 100%. At this scale you are not viewing the image at a realistic size -- even when represented on a large monitor or print, noise that is obvious at high zoom factors will be virtually invisible on the final image. However, some modules further along the image pipeline (especially those that increase local contrast) may end up exaggerating any noise that is present so, again, there are trade-offs to be made.
+
+The first module you can use to manage this is [_demosaic_](../../module-reference/processing-modules/demosaic.md), which controls how the single-color (R, G or B) pixels in your Raw file are converted to image pixels combining all three colors. Each demosaic algorithm has its own trade-offs between retaining fine detail and reducing noise. The default demosaic algorithm (RCD) usually provides a reasonable compromise.
+
+Demosaic algorithms can only do so much to manage noise in your image. The [_denoise (profiled)_](../../module-reference/processing-modules/denoise-profiled.md) module is individually tuned for a number of common camera sensors and can be used to reduce or remove pixel noise. As with demosaic you should alter the settings until you are happy with the balance of denoising vs fine-detail reproduction. The default settings are usually sufficient.
+
+### sharpness and local contrast
+
+A number of modules can be used to adjust the local contrast and sharpness of your image. Most of these modules aim to enhance the apparent contrast of edges and do not add "real" sharpness (they are not the same as lens deconvolution). You should take care when using these modules as most of them can introduce artifacts (such as halos) when settings are pushed too far:
+
+- the [_contrast equalizer_](../../module-reference/processing-modules/contrast-equalizer.md) module allows you to adjust contrast, limiting the effect to certain feature sizes. For example you can use it to increase the contrast of fine details without impacting larger-scale objects, or vice versa
+- the [_diffuse or sharpen_](../../module-reference/processing-modules/diffuse.md) module offers a number of presets for sharpening, lens deblurring and the addition of local contrast
```

Comment:
The heading at the start of this list says that these modules are affecting local contrast rather than doing a deconvolution. However, the diffuse or sharpen module doesn't really fit in that category. Maybe it should be mentioned separately to this list since it works differently to the others.

Reply:
I'm trying to do task driven stuff here so it's about increasing sharpness and/or local contrast. I know diffuse/sharpen can do other things as well but it does do this. As I understood it diffuse/sharpen wasn't deconvolution either.